src/components/pacbio/PacbioRunPlate.vue

```diff
@@ -73,6 +82,10 @@ export default {
     onWellClick(position, plateNumber) {
       this.$refs.modal.showModalForPositionAndPlate(position, plateNumber)
     },
+    // Only used for Revio runs
+    serialNumber(plateNumber) {
```

Comment:
What I am wondering is if we make the plate itself computed then the serial number can also be computed so you don't also have to retrieve the plate again? so serial number would just be `this.plate.sequencing_kit_box_barcode.substring(15,20)`

Reply:
Ye I wondered about making it computed, but both serialNumber/ plate computed would need to be passed the plateNumber, and didn't think you can pass in args to computed?